draft-irtf-cfrg-vdaf.md

```diff
@@ -4508,12 +4507,11 @@ picks bogus measurements for the remaining Clients.  Applications can guard
 against these risks by adding additional controls on report submission,
 such as Client authentication and rate limits.
 
-VDAFs do not inherently provide differential privacy {{Dwo06}}.  The VDAF approach
-to private measurement can be viewed as complementary to differential privacy,
-relying on non-collusion instead of statistical noise to protect the privacy of
-the inputs.  It is possible that a future VDAF could incorporate differential
-privacy features, e.g., by injecting noise before the sharding stage and
-removing it after unsharding.
+VDAFs do not inherently provide differential privacy {{Dwo06}}: depending on
+the distribution of the measurements, the aggregate result itself can still
+leak a significant amount of information about the measurements. Thus
+applications SHOULD execute a VDAF with a compatible mechanism for differential
```

Comment:
```suggestion
applications might compose a VDAF with a compatible mechanism for differential
```

Reply:
I deleted the sentence per @ekr's suggestion ... I can add it back with "might" if you like.